src/components/LayoutModules/ListItemModule/ListItemModule.stories.js

```diff
@@ -0,0 +1,231 @@
+/**
+ * @file List item module stories.
+ * @copyright IBM Security 2021
+ */
+
+import {
+  Bee16,
+  InProgress16,
+  Locked16,
+  UserAvatar20,
+} from '@carbon/icons-react';
+
+import { action } from '@storybook/addon-actions';
+
+import React from 'react';
+
+import { getDocsParameters } from '../../../../.storybook';
+import withResponsive from '../../../../.storybook/decorators';
+
+import { ICA, ListItemModule, ProfileImage, StatusIcon, Tag } from '../../..';
+
+import getTitle from '../stories';
+import page from './index.mdx';
+
+export default {
+  title: getTitle({
+    displayName: 'ListItemModule (Canary)',
+    name: 'ListItemModule',
+  }),
+  component: ListItemModule,
+  parameters: {
+    docs: { page },
+
+    ...getDocsParameters(),
+  },
+  decorators: [withResponsive],
+};
+
+export const Default = () => (
+  <ListItemModule href="#">
+    {({ Column, getLayoutProps }) => (
+      <>
+        <Column>
+          <Bee16 {...getLayoutProps({ type: 'icon' })} />
```

Comment:
I've been thinking a lot about the API for this recently - what are your thoughts on `getLayoutProps` returning an object of props to choose from instead of the consumer passing a `type` parameter? For example, this line becomes `<Bee16 {...getLayoutProps().icon} />` or for multiple styles, it could become:

```
<ListItemModule href="#">
  {({ Column, getLayoutProps }) => {
    const { icon, title } = getLayoutProps();      

    return (
      <>
        <Bee16 {...icon} />
        <h2 {...title}>List title</h2>
          ...

Reply:
I prefer `const { icon, title } = getLayoutProps();` as it's a more common pattern than `{...getLayoutProps({ type: 'icon' })}`, but just looking at `<Bee16 {...icon} />` doesn't tell me how much prop spreading is actually happening. If I see `{...props}` then I expect they are generic props passed down from the parent (width, height, etc), but `{...icon}` is both specific and vague at the same time.

Reply:
Hey guys, I agree that looks cleaner on the consumer side. It adds a bit of bulk to our `ListItemModule` itself but it's nice to see all the different options in one place. Just pushed a new commit with new preview link, let me know what you all think.